src/utils/inference.py

```diff
@@ -73,3 +73,9 @@ def get_standing_laying_prediction_local(numpy_array):
 
 def get_depthmapmultiartifactlatefusion_height_predictions_local(numpy_array):
     return depthmapmultiartifactlatefusion_height_model.predict(numpy_array)
+
+
+def get_ensemble_height_predictions_local(model_path, numpy_array):
+    model_path += '/outputs/best_model.ckpt/'
```

Comment:
I don't think code is correct.
If _model_path_ is args of _get_ensemble_height_predictions_local()_, 
why you are harcoding value in line 79 
_model_path += '/outputs/best_model.ckpt/'_

Reply:
Missed the + sign.
Resolved !!!